mainline-alpine/drupal.conf

```diff
@@ -1,5 +1,5 @@
 server {
-    server_name _;
+    server_name SERVER_NAME;
     root /var/www/html;
```

Comment:
IMHO we should tokenize `root` as well. I suggest name nginx independent so I could implement it the same way in drupal-docker/apache (maybe `DOCROOT`).

```
root DOCROOT
```

Probably even better to use before/after special char, something like `%SERVER_NAME%` and `%DOCROOT%`